src/main/java/org/cryptimeleon/craco/sig/sps/kpw15/SPSKPW15SigningKey.java

```diff
@@ -89,18 +88,10 @@ public void setK(Zp.ZpElement[] K) {
         return P0;
```

Comment:
Mention in the class JavaDoc that you opted to stick closely to the notation in the paper. Therefore they are capitalised (K, P0 and so on). This is not Java standard but better to stick close to the paper.

Reply:
[50cd892](https://github.com/Colamaleon/craco/pull/2/commits/50cd8925bfec3139f3e0de29bc1c3bd81c5f86d5)

Reply:
as above